docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -271,3 +271,7 @@ There are now generalized events for updating (`UPDATE_CREDENTIAL`) and removing
 The new event types are supported by the Email Event Listener.
 
 The following event types are now deprecated and will be removed in a future version: `UPDATE_PASSWORD`, `UPDATE_PASSWORD_ERROR`, `UPDATE_TOTP`, `UPDATE_TOTP_ERROR`, `REMOVE_TOTP`, `REMOVE_TOTP_ERROR`
+
+= `--import-realms` option can import the master realm
+
+When running the a start command with the `--import-realms` option before the master realm exists, it will be imported if it exists in the import material. The previous behavior was that the master realm was created first, then its import skipped. 
```

Comment:
```suggestion
= `--import-realm` option can import the master realm

When running the `start` command with the `--import-realms` option before the master realm exists, it will be imported if it exists in the import material. The previous behavior was that the master realm was created first, then its import skipped. 
```

Reply:
I'm using the term a start command to cover both start and start-dev - do you prefer to ignore the start-dev case?

Reply:
Ah, right. Maybe would be better to be more explicit? And using something like 'when running the `start` (or `start-dev`) command with the ....'

But the `a start command ...` works for me as well.

And the `--import-realms` should be replaced by `--import-realm` everywhere - if I'm not mistaken.

Reply:
should be corrected